Parser.fs

```diff
@@ -58,22 +58,35 @@ let parseIdentifier =
 
 // Bracket parsers.
 
-/// Parser for items in a pair of matching brackets.
-/// Automatically parses any whitespace after `bra`, and before `ket`.
+/// <summary>
+///     Parser for items in a pair of matching brackets.
+///     Automatically parses any whitespace after `bra`, and before `ket`.
+/// </summary>
```

Comment:
Lots of boring changes to top-level comments here, but there are important parser changes coming.